setup.cfg

```diff
@@ -0,0 +1,26 @@
+[bumpversion]
+current_version = 0.1.0
+commit = True
+tag = True
+
+[bumpversion:file:setup.py]
+search = version='{current_version}'
+replace = version='{new_version}'
+
+[bumpversion:file:stockroom/__init__.py]
+search = __version__ = '{current_version}'
+replace = __version__ = '{new_version}'
+
+[bdist_wheel]
+universal = 1
```

Comment:
This should be set to `universal = 0` since it is not compatible with `python 2`

Reply:
nice! thanks for figuring out that for me

Reply:
Yup. I know. It's in my to-do list ..